src/Classes/Controller/RegisterUserController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: academy
+ * Date: 15/05/2018
+ * Time: 16:37
+ */
+
+namespace Portal\Controller;
+
+use Portal\Models\UserModel;
+use Slim\Http\Request;
+use Slim\Http\Response;
+
+class RegisterUserController
+{
+    private $userModel;
+
+    /**
+     * Instantiates RegisterUserController
+     *
+     * @param UserModel $userModel userModel object dependency
+     */
+    public function __construct(UserModel $userModel)
+    {
+        $this->userModel = $userModel;
+    }
+
+    /**
+     * Validates username / email and updates database if email doesn't exist
+     *
+     * @param Request $request HTTP request
+     * @param Response $response HTTP response
+     *
+     * @return error/success message and status code
+     */
+    function __invoke(Request $request, Response $response){
+
```

Comment:
This controller doesn't check `$_SESSION['loggedIn']`  so you could post a User using `curl` or Postman without being logged in